### ISSUE TITLE
Formatting of Incubator.ChipsInput wrong if on more than one line.

### ISSUE THREAD

**Issue by btrewern** [wix/react-native-ui-lib#2086]
## Description

If there are more than one line of chips on the Incubator.ChipsInput input the chips overlap with the component below.

### Related to

- [*] Components
- [ ] Demo
- [ ] Docs
- [ ] Typings

### Steps to reproduce

Create a new expo app.
Add / replace the following code:

```
import { useState } from 'react';
import { StyleSheet, View } from 'react-native';
import { Button, Incubator } from 'react-native-ui-lib';

export default function App() {
  const [chips, setChips] = useState([]);
  return (
    <View style={styles.container}>
      <Incubator.ChipsInput
        chips={chips}
        onChange={(chips) => setChips(chips)}
        />
      <Button label='Press Me' onPress={() => alert('test')}/>
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    margin: 20,
    padding: 10,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
```

#### Expected behavior

The height of the control should contain all the chips.

#### Actual behavior

The chips overflow under the component below.

### Screenshots/Video

![Screenshot_2022-06-13-20-19-39-759_host exp exponent](https://user-images.githubusercontent.com/274824/173428957-00a629c4-a4b1-4905-bb94-e9dc46e4d3dd.jpg)

### Environment

- Expo-cli 5.4.9
- React Native UI Lib: 6.17.1

### Affected platforms

- [*] Android
- [ ] iOS
- [ ] Web

**Comment by ethanshar** [comment]
Hi @btrewern 
I believe this issue was fixed in this PR #2183
Can you please verify and reopen the ticket if needed

**Comment by btrewern** [comment]
Looks fixed to me.  Thanks.
